spec/models/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require './app/models/user'
+
+describe User do
+
+  let!(:user){create :user}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.